tsheets/results.py

```diff
@@ -9,6 +9,8 @@ def __init__(self, url, options, repo, bridge, is_singleton=False, mode="list"):
         self.repo = repo
         self.model = repo.model
         self.name = class_to_endpoint(self.model.__name__) + ('' if is_singleton else 's')
+        if isinstance(self.repo, repos.TimesheetsDeleted):
+            self.name = 'timesheets_deleted'
```

Comment:
What you have here works, but wondering if it is better to create a method in `repository.py` to set the name.
We can avoid one-off logic like this, and also if new endpoints are added that follow `timesheets_deleted` pattern, then we wouldn't need to update the logic here again.
Thoughts?